externs/custom-elements.js

```diff
@@ -62,3 +62,6 @@ Element.prototype.__CE_definition;
 
 /** @type {!DocumentFragment|undefined} */
 Element.prototype.__CE_shadowRoot;
+
+/** @type {boolean} */
+HTMLElement.es5Shimmed;
```

Comment:
Why do we need this extern, given that the native shim isn't meant to be compiled? Assuming we do need it, can you prefix it with `__CE_`?

Reply:
```suggestion
HTMLElement.__CE_es5Shimmed;
```